### PR TITLE
finished signin page draft, no link/router or user authentication

### PR DESCRIPTION
**Pull request by jxu710**
A draft signin Page for review , please open in smaller screen in dev tool

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,31 +1,120 @@
...
+
+          <IonRow>
+            <IonCol>
+              <h5 className="ion-text-center">
+                Already a User? <a href="#"> Login.</a>
+              </h5>
+            </IonCol>
+          </IonRow>
+        </IonGrid>
+      </IonContent>
```
Comment (cwaku) [inline src/App.tsx]: - [ ] For code reusability and also to conform to the [S.O.L.I.D](https://www.youtube.com/watch?v=MSq_DCRxOxw&t=668s) principle, I recommend splitting this into its component and then reusing it in the `App.tsx`. Preferably create a `components` folder to keep this and future components in there.

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @jxu710 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/App.tsx (hunk 1 above)
Comment (jxu710) [reply src/App.tsx]: splitted into components, thank you

**Review by cwaku — APPROVED** [review]
Hi @jxu710 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------

**Pull request merged**